pandas/io/tests/parser/common.py

```diff
@@ -1467,5 +1469,23 @@ def test_memory_map(self):
             'c': ['I', 'II', 'III']
         })
 
-        out = self.read_csv(mmap_file, memory_map=True)
-        tm.assert_frame_equal(out, expected)
+        result = self.read_csv(mmap_file, memory_map=True)
+        tm.assert_frame_equal(result, expected)
+
+    def test_read_csv_utf_aliases(self):
+        # see gh issue 13549
+        path = 'test.csv'
+        expected = DataFrame({'A': [0, 1], 'B': [2, 3],
+                              'multibyte_test': ['testing123', 'bananabis'],
+                              'mb_nums': [154.868, 457.8798]})
+        with tm.ensure_clean(path) as path:
+            for byte in [8, 16]:
+                expected.to_csv(path, encoding='utf-' + str(byte), index=False)
+                for fmt in ['utf-{0}', 'utf_{0}', 'UTF-{0}', 'UTF_{0}']:
+                    encoding = fmt.format(byte)
+                    for engine in ['c', 'python', None]:
```

Comment:
This is not necessary (nor is the `engine` keyword).  The test suite will cover both engines (and the default case is not needed here).  That's why it's `self.read_csv` and not `read_csv`.  You are in fact running the test suite TWICE for each engine.